src/usage_metrics/jobs/datasette.py

```diff
@@ -39,13 +24,32 @@ def datasette_weekly_partition(start: datetime, end: datetime):
     }
 
 
-@job(
-    config=datasette_weekly_partition,
+@graph
+def transform(raw_logs: pd.DataFrame) -> pd.DataFrame:
+    """Transform datasette logs."""
+    df = da.unpack_httprequests(raw_logs)
+    df = da.parse_urls(df)
+    return da.geocode_ips(df)
+
+
+@graph
+def process_datasette_logs():
+    """Process datasette logs locally using a SQLite database."""
```

Comment:
I believe this comment should be updated to remove locally

Reply:
Ah thanks